website/blog/2021-05-31-tungsten.md

```diff
@@ -0,0 +1,102 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.10.4 - Tungsten
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.10.4, which adds support for Scala 2.12.14. 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">24</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">11</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">4</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
```

Comment:
I usually check the last milestone, add any issues that we closed after the last release and use the number there, so for example here: 
https://github.com/scalameta/metals/milestone/39

```suggestion
    <td align="center">3</td>
```